static/js/components/search/CustomResetFiltersDisplay_test.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React from 'react';
+import _ from 'lodash';
+import { shallow } from 'enzyme';
+import { assert } from 'chai';
+
+import CustomResetFiltersDisplay from './CustomResetFiltersDisplay';
+
+describe('CustomResetFiltersDisplay', () => {
+  let props = {
+    clearAllLabel: "Clear all filters",
+    hasFilters: true,
+    resetFilters: () => {},
+    bemBlock: (): Object => {
+      return {
+        state: (): void => {}
+      };
+    }
+  };
+
+  it('renders reset filters link', () => {
+    const wrapper = shallow(<CustomResetFiltersDisplay {...props}/>);
+
+    assert.equal(wrapper.children().children().text(), 'Clear all filters');
+  });
+
+  it('assert reset filter link does not render', () => {
```

Comment:
can you make this message more explicit? something like `"reset filter link does not render when hasFilters is false"`